apps/frontend/src/lib/storage/types/user/connection/index.ts

```diff
@@ -1,19 +1,22 @@
 import { z } from "zod";
+import { nullToUndefined } from "@types";
 import { UserDataSchema } from "../userData";
 import { CommentDataSchema } from "./commentData";
 import { TapDataSchema } from "./tapData";
 import { PSIDataSchema } from "./psiData";
-import { nullToUndefined } from "@types";
+import { SentMessageSchema } from "./sentMessage";
 
 export const ConnectionSchema = z.object({
   user: UserDataSchema,
   comment: nullToUndefined(CommentDataSchema),
   taps: z.array(TapDataSchema),
   psi: nullToUndefined(PSIDataSchema),
+  sentMessages: nullToUndefined(z.array(SentMessageSchema)).default([]),
```

Comment:
Just to make sure I understand -- there's no need for receivedMessages because the "tapper" already has the connection's information, and there's no other message types (yet) which require it?

Reply:
yes. received messages always get parsed into the Connection struct, whether its user data or taps